pulseaudio-control.bash

```diff
@@ -320,6 +348,30 @@ function usage() {
     echo "    https://github.com/marioortizmanero/polybar-pulseaudio-control"
 }
 
+while [[ "$1" = --* ]]; do
+    case "$1" in
+        --vol-icon-low=*)
+            VOLUME_ICONS[0]="${1#--vol-icon-low=}"
+            ;;
+        --vol-icon-mid=*)
+            VOLUME_ICONS[1]="${1#--vol-icon-mid=}"
+            ;;
+        --vol-icon-high=*)
+            VOLUME_ICONS[2]="${1#--vol-icon-high=}"
+            ;;
+        --sink-icon=*)
+            SINK_ICON="${1#--sink-icon=}"
+            ;;
+        --sink-name-from=*)
+            setNicknames "${1#--sink-name-from=}"
```

Comment:
I don't understand why you need `--sink-name-from`. I was expecting something as simple as `--sink-nickname "nick=id"` or similars.

Reply:
Yes I didn't mean to replace it absolutely - this was just a good enough first pass for my own use, coupled with I don't really want to hard code an ID -> nick mapping, personally. (I'd prefer to setup on a new computer, or with a new device, and still have something sane without fiddling again.)

`device.description` was just an example here, `--sink-name-from` as implemented should work with any property, and if not given (or for devices without that property) the existing nickname mapping will still be used.

> I also wouldn't want to modify any pulseaudio configuration inside this script, it's meant to be "read-only" to avoid any issues. I'd expect the user to change them outside the script, if anything.

Yes absolutely :+1: - that's what I meant - that `device.description` or even some custom `polybar.nick` could be set outside of this script, and then you run `--sink-name-from=<whatever-prop>` knowing that that's been configured elsewhere.